bots/hermes/config.ini

```diff
@@ -0,0 +1,2 @@
+[Telegram]
+token=1867217803:AAHIHvwS8FZ7LXHbXTASWlSc9CJNmW_GlXU
```

Comment:
Is this a real token ? If yes you should deactivate it, and remove that token from the file.
The right side of this equality should be empty